tests/test-dladdr.c

```diff
@@ -112,11 +112,13 @@ int check_dladdr_by_dlopen( char *libname, char *sym, int should_match )
 #define DWORD long
 #define SIZE_T long
 
+#ifndef _MSC_VER
 /* link to import thunk */
 HMODULE __stdcall GetModuleHandleA (LPCSTR lpModuleName);
+SIZE_T __stdcall VirtualQuery (LPCVOID lpAddress, LPCVOID lpBuffer, SIZE_T dwLength);
```

Comment:
You cannot hide VirtualQuery when compiling under msvc. Otherwise compilation on msvc fails as it cannot find VirtualQuery function.

Reply:
Also tests on Appveyor due to this issue failed.